nixos/modules/security/pam.nix

```diff
@@ -295,6 +295,8 @@ let
       };
 
       limits = mkOption {
+        default = [];
+        type = limitsType;
         description = ''
           Attribute set describing resource limits.  Defaults to the
           value of <option>security.pam.loginLimits</option>.
```

Comment:
We might want to have a link to some docs here, so the choices below make a bit more sense.